src/lib/y2storage/partition.rb

```diff
@@ -126,27 +126,51 @@ def disk
       partitionable.is_a?(Disk) ? partitionable : nil
     end
 
-    # Whether the first sector of the partition is aligned according to
+    # Position of the first block of the region
+    #
+    # @return [Integer]
+    def start
+      region.start
+    end
+
+    # Position of the last block of the region
+    #
+    # @return [Integer]
+    def end
+      region.end
+    end
+
+    # Size of a single block
+    #
+    # @return [DiskSize]
+    def block_size
+      region.block_size
+    end
```

Comment:
Shouldn't `#start`, `#end` and `#block_size` be defined in `BlkDevice` instead of this subclass? They apply to any region-based device.

Reply:
I was sure that I was adding it to `BlkDevice` :S